src/lib/country-data.js

```diff
@@ -26,3 +59,66 @@ module.exports.subdivisionOptions = Object.keys(countries).reduce((subByCountry,
         });
     return subByCountry;
 }, {});
+
+/**
+ * Take startingCountryOptions, and override any number of its country labels.
+ * E.g., to use the German and Italian names of Germany and Italy, do:
+ *     overrideLabels({de: 'Deutschland', it: 'Italia'})
+ * @param {array} startingCountryOptions array of country objects
+ *     like [{value: 'ca', label: 'Canada'}, ...]
+ * @param {object} countryNameOverrides map of country codes we should override,
+ *     to labels to use instead of default
+ * @returns {array} revised array of country objects, with overridden labels
+ */
+const overrideLabels = module.exports.overrideLabels = (startingCountryOptions, countryNameOverrides) => {
+    // shallow copy two levels deep, since that's as low as countryOptions goes
+    const newCountryOptions = startingCountryOptions.map(item => ({...item}));
+    Object.keys(countryNameOverrides).forEach(codeToOverride => {
+        const existing = newCountryOptions
+            .find(countryOption => countryOption.value === codeToOverride.toLowerCase());
+        if (existing) existing.label = countryNameOverrides[codeToOverride];
+    });
+    return newCountryOptions.sort(orderByLabel);
+};
+
+/**
+ * Function dupeCommonOptions():
+ * takes startingCountryOptions, and duplicates any number of its country labels
+ * at the beginning of the array.
+ * E.g., to have "Germany" and "Italy" at the top of the list, use:
+ *     dupeCommonOptions(['de', 'it')
+ * @param {array} startingCountryOptions array of country objects
+ *     like [{value: 'ca', label: 'Canada'}, ...]
+ * @param {array} commonCountryCodes array of country codes we should duplicate
+ * @returns {array} revised array of country objects, with duplicates inserted at beginning
+ */
+const dupeCommonOptions = module.exports.dupeCommonOptions = (startingCountryOptions, commonCountryCodes) => {
```

Comment:
Same comment on this one about exporting.

Reply:
it needs to be exported to be unit tested.